stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -140,18 +140,16 @@ private ClientCalls() {}
    * Returns the result of calling {@link Future#get()} interruptably on a task known not to throw a
    * checked exception.
    *
-   * <p>If interrupted, the interrupt is restored before throwing a {@code RuntimeException}.
+   * <p>If interrupted, the interrupt is restored before throwing an exception..
    *
-   * @throws RuntimeException if {@code get} is interrupted
-   * @throws CancellationException if {@code get} throws a {@code CancellationException}
```

Comment:
Could we leave the CancellationException line, because that is still possible? I am certain it doesn't matter to any of the callers, but this method itself doesn't know if cancellation is a problem or not.